sentry-android-navigation/src/main/java/io/sentry/android/navigation/SentryNavigationListener.kt

```diff
@@ -140,6 +142,8 @@ class SentryNavigationListener @JvmOverloads constructor(
             transactonOptions
         )
 
+        transaction.spanContext.origin = TRACE_ORIGIN
```

Comment:
`h`: we'd also want to track if it was `auto.navigation.jetpack_compose`. For that I guess we could just add a new optional ctor parameter and then set it from [SentryLifecycleObserver](https://github.com/getsentry/sentry-java/blob/f274c795d1c562858cf44f547b8c1e69bc2e0bee/sentry-compose/src/androidMain/kotlin/io/sentry/compose/SentryNavigationIntegration.kt#L23-L24) and concat here afterwards